src/logic/azext_logic/generated/custom.py

```diff
@@ -0,0 +1,186 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+import json
+from knack.util import CLIError
+
+
+def logic_workflow_list(cmd, client,
+                        resource_group_name=None,
+                        top=None,
+                        filter=None):
+    if resource_group_name:
+        return client.list_by_resource_group(resource_group_name=resource_group_name,
+                                             top=top,
+                                             filter=filter)
+    return client.list_by_subscription(top=top,
+                                       filter=filter)
+
+
+def logic_workflow_show(cmd, client,
+                        resource_group_name,
+                        name):
+    return client.get(resource_group_name=resource_group_name,
+                      workflow_name=name)
+
+
+def logic_workflow_create(cmd, client,
+                          resource_group_name,
+                          name,
+                          definition,
+                          location,
+                          tags=None,
+                          state=None,
+                          endpoints_configuration=None,
+                          access_control=None,
+                          integration_account=None,
+                          integration_service_environment=None):
+    with open(definition) as json_file:
```

Comment:
We can use `type=validate_file_or_dict` to handle json file automatically. You can reference [here](https://github.com/Azure/azure-cli-extensions/blob/27238a67ce52ac497ed2813d01f5eb49c8260788/src/stream-analytics/azext_stream_analytics/_params.py#L84).

Reply:
Added